api/controllers/CampaignBotController.js

```diff
@@ -70,30 +67,33 @@ CampaignBotController.prototype.chatbot = function(req, res) {
       req.incoming_image_url);
   }
 
-  dbUsers.findOne({ '_id': req.user_id }, function (err, userDoc) {
-
-    if (err) {
-      return self.handleError(req, res, err);
-    }
+  this.loadUserAndSignup(req, res);
+}
 
+/** 
+ * Creates user and posts Signup to the Campaign.
+ * @param {object} req - Express request
+ * @param {object} res - Express response
+ */
+CampaignBotController.prototype.loadUserAndSignup = function(req, res) {
+  logger.debug('%s loadUserAndSignup', this.loggerPrefix(req));
+  const promise = dbUsers.findOne({ '_id': req.user_id }).exec();
```

Comment:
Can we make it more [descriptive](https://github.com/airbnb/javascript#naming--descriptive) (like `userPromise`) or just chain `then()`?